rskj-core/src/test/java/co/rsk/peg/BridgeSupportIT.java

```diff
@@ -155,88 +174,151 @@ void setUpOnEachTest() {
             LockingCapMainNetConstants.getInstance(),
             signatureCache
         );
+
+        track = createRepository().startTracking();
+        arrangeBridgeSupport(bridgeRegTestConstants, activations);
```

Comment:
i insist, don't do this. we don't want a method for arranging the bridge support, that's why we created the bridge support builder.

Reply:
btw, can we use mainnet for general setup?

Reply:
What's your proposal for avoid recreating the bridge support when we need an specific BridgeSupport on certain test cases?

Reply:
Done. Got rid `arrangeBridgeSupport`.